yarn.lock

```diff
@@ -0,0 +1,4196 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Please don't include lock files in utilities like sanitize-html

Reply:
Oops, forgot to delete